File: ce/customerengagement/on-premises/snippets/csharp/CRMV8/businessmanagement/cs/winquote.cs
// <snippetwinquote>



 // Mark the quote as won
 // Note: this is necessary in order to convert a quote into a 
 // SalesOrder.
 WinQuoteRequest winQuoteRequest = new WinQuoteRequest()
 {
     QuoteClose = new QuoteClose()
     {
         Subject = "Quote Close" + DateTime.Now.ToString(),
         QuoteId = quote.ToEntityReference()
     },
     Status = new OptionSetValue(-1)
 };
 _serviceProxy.Execute(winQuoteRequest);

 Console.WriteLine("Quote won.");

// </snippetwinquote>

File: ce/customerengagement/on-premises/snippets/visualbasic/CRMV8/businessmanagementvb/vb/winquote.vb
' <snippetwinquote>


 ' Mark the quote as won
 ' Note: this is necessary in order to convert a quote into a 
 ' SalesOrder.
 Dim winQuoteRequest_Renamed As New WinQuoteRequest() With
     {
         .QuoteClose =
         New QuoteClose() With
         {
             .Subject = "Quote Close" &amp; Date.Now.ToString(),
             .QuoteId = quote_Renamed.ToEntityReference()
         },
         .Status = New OptionSetValue(-1)
     }
 _serviceProxy.Execute(winQuoteRequest_Renamed)

 Console.WriteLine("Quote won.")

' </snippetwinquote>